prosody/Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN \
 
 FROM ${JITSI_REPO}/base
 
+ENV XMPP_CROSS_DOMAIN="false"
```

Comment:
We tend not to do this, why do it for this specific value?

Reply:
I think we should start defining the defaults in the dockerfile instead of defining defaults all over again in several files. There should be a single point of truth which should always be the env var IMO. I think this PR can be a start for this transition.

Reply:
Fair enough!